packages/server/src/scripts/create_tenant.js

```diff
@@ -3,6 +3,7 @@ require('dotenv').config();
 const inquirer = require('inquirer');
 const { validate: validateEmail } = require('email-validator');
 const knex = require('../db/connection');
+const { seedReportingPeriods, seedApplicationSettings } = require('../../seeds/dev/02_backfill_arpa_reporting_periods');
```

Comment:
I think I would rather import from a script to seed file than the reverse, but that's a super easy refactor.  No reason to hold up merging.